scripts/5min/Plot_ten_systems_one_year.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+# Select and plot one year of PV data from ten PV systems
+
+import pandas as pd
+import matplotlib.pyplot as plt
+
+# Extract data of 10 PV systems
+PV_data = pd.DataFrame(columns = ["generation_wh","timestamp","ss_id"])
+PV_systems = [13308, 13057, 7548, 7542, 10589, 6891, 9369, 11438, 18989, 7243]
+for i in range(10): 
+    PV = PV_systems[i]    
+    PV_data = PV_data.append(data.loc[data['ss_id'] == PV])
+    
```

Comment:
I think you could use `PV_data = data[data['ss_id'].isin(PV_systems)]` to get all the systems in one go